src/PointwiseFunctions/AnalyticSolutions/Elasticity/HalfSpaceMirror.cpp

```diff
@@ -2,234 +2,180 @@
+      get<0, 2>(strain)[i] = cos_phi * strain_rz;
+      get<1, 1>(strain)[i] =
+          strain_pp + square(sin_phi) * (strain_rr - strain_pp);
+      get<1, 2>(strain)[i] = sin_phi * strain_rz;
+      get<2, 2>(strain)[i] = strain_zz;
+
     } else {
-      auto displacement_w = integration(
-          [&r, &z, &shear_modulus, &lame_parameter, this](const double k) {
-            return applied_force_ / (2. * shear_modulus) *
-                   gsl_sf_bessel_J1(k * r) * exp(-k * z) *
-                   (1 -
-                    (lame_parameter + 2. * shear_modulus) /
-                        (lame_parameter + shear_modulus) +
-                    k * z) *
-                   1 / (2. * M_PI) * exp(-square(k * beam_width_ / 2.));
-          },
-          lower_boundary, absolute_tolerance_);
-
-      cos_phi = get<0>(x)[i] / w[i];
-      sin_phi = get<1>(x)[i] / w[i];
-      strain_pp = displacement_w / w[i];
-      strain_rr = theta - strain_pp - strain_zz;
+      get<0, 0>(strain)[i] = 0.5 * (trace_term - strain_zz);
+      get<1, 1>(strain)[i] = 0.5 * (trace_term - strain_zz);
+      get<2, 2>(strain)[i] = strain_zz;
     }
-    get<0, 0>(strain)[i] =
-        strain_pp + square(cos_phi) * (strain_rr - strain_pp);
-    get<0, 1>(strain)[i] = cos_phi * sin_phi * (strain_rr - strain_pp);
-    get<0, 2>(strain)[i] = cos_phi * strain_rz;
-    get<1, 1>(strain)[i] =
-        strain_pp + square(sin_phi) * (strain_rr - strain_pp);
-    get<1, 2>(strain)[i] = sin_phi * strain_rz;
-    get<1, 0>(strain)[i] = get<0, 1>(strain)[i];
-    get<2, 0>(strain)[i] = get<0, 2>(strain)[i];
-    get<2, 1>(strain)[i] = get<1, 2>(strain)[i];
-    get<2, 2>(strain)[i] = strain_zz;
   }
   return {std::move(strain)};
 }
 
-tuples::TaggedTuple<::Tags::FixedSource<Tags::Displacement<dim>>>
-HalfSpaceMirror::variables(
-    const tnsr::I<DataVector, dim>& x,
-    tmpl::list<
-        ::Tags::FixedSource<Tags::Displacement<dim>>> /*meta*/) noexcept {
-  return {make_with_value<tnsr::I<DataVector, dim>>(x, 0.)};
-}
-
-tuples::TaggedTuple<::Tags::Initial<Tags::Displacement<dim>>>
-HalfSpaceMirror::variables(
-    const tnsr::I<DataVector, dim>& x,
-    tmpl::list<::Tags::Initial<Tags::Displacement<dim>>> /*meta*/) noexcept {
-  return {make_with_value<tnsr::I<DataVector, dim>>(x, 0.)};
-}
-
-tuples::TaggedTuple<::Tags::Initial<Tags::Strain<dim>>>
+tuples::TaggedTuple<::Tags::FixedSource<Tags::Displacement<3>>>
 HalfSpaceMirror::variables(
-    const tnsr::I<DataVector, dim>& x,
-    tmpl::list<::Tags::Initial<Tags::Strain<dim>>> /*meta*/) noexcept {
-  return {make_with_value<tnsr::ii<DataVector, dim>>(x, 0.)};
+    const tnsr::I<DataVector, 3>& x,
+    tmpl::list<::Tags::FixedSource<Tags::Displacement<3>>> /*meta*/) noexcept {
+  return {make_with_value<tnsr::I<DataVector, 3>>(x, 0.)};
 }
 
 void HalfSpaceMirror::pup(PUP::er& p) noexcept {
   p | beam_width_;
-  p | applied_force_;
   p | constitutive_relation_;
```

Comment:
serialize all member variables, including those with default values in the constructor